movie/urls.py

```diff
@@ -18,10 +18,10 @@
 from django.urls import path, include
 from django.conf.urls.static import static
 from django.conf import settings
-
+from user.views import home
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('user/', include(('user.urls', 'user'), namespace='user')),
-    
+    path('', home, name='home'),
```

Comment:
why are the urls changed? the url for home should be /user/

Reply:
i thought the "/" url is missing

Reply:
- first it will try to access only http://127.0.0.1:8000/ route

Reply:
i want the http://127.0.0.1:8000/user/shows/ to be valid and not http://127.0.0.1:8000/

Reply:
should i redirect it to second route but will then also hit http://127.0.0.1:8000/ route then get redirected

Reply:
no leave the url logic intact

Reply:
means i don't need to change this ??

Reply:
so what will be there when the http://127.0.0.1:8000/  gets hit ?
do i need to redirect ?

Reply:
no thats a separate issue. No need to worry about that